public/app/features/dashboard/components/SaveDashboard/useDashboardSave.tsx

```diff
@@ -36,6 +36,9 @@ export const useDashboardSave = (dashboard: DashboardModel) => {
 
   const notifyApp = useAppNotification();
   useEffect(() => {
+    if (state.error) {
```

Comment:
This seemed surprising that no error was shown before... so not sure what the best course of action is here